src/options.cpp

```diff
@@ -1988,6 +1988,8 @@ bool options_manager::save()
     message_ttl = ::get_option<int>( "MESSAGE_TTL" );
     fov_3d = ::get_option<bool>( "FOV_3D" );
 
+    update_music_volume();
```

Comment:
If you're only invoking this with no arguments, there is no need for the overload with an argument.